### PR TITLE
Increase retry interval for ipywidgets tests

### PR DESCRIPTION
**Pull request by joyceerhl**
For https://github.com/microsoft/vscode-jupyter/issues/3955, https://github.com/microsoft/vscode-jupyter/issues/238

bqplot sometimes doesn't load because the internet connection appears to be down. Not sure why this doesn't impact the other ipywidgets, but I can repro the failure locally (it's flaky, sometimes passes and sometimes fails). Bump the timeout to see if that makes a difference.

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Review by DonJayamanne — APPROVED** [review]

**Review by IanMatthewHuff — APPROVED** [review]

**Review by DavidKutu — APPROVED** [review]

File: src/test/datascience/uiTests/ipywidget.ui.functional.test.ts [hunk 1]
```diff
@@ -220,20 +220,19 @@ use(chaiAsPromised);
...
             await retryIfFail(async () => {
+                await notebookUI.executeCell(1);
+                await notebookUI.executeCell(3);
                 const cellOutput = await notebookUI.getCellOutputHTML(3);
 
                 // Confirm cells with output has been rendered.
                 assert.include(cellOutput, 'Hello</td>');
                 assert.include(cellOutput, 'World</td>');
-            });
+            }).catch(done);
```
Comment (DonJayamanne) [inline src/test/datascience/uiTests/ipywidget.ui.functional.test.ts:235]: This won't make any difference.

`done` is meant to be used without the `async` keyword.
Old style async code so we can tell the calling code via a callback when the function is done.

Comment (joyceerhl) [reply src/test/datascience/uiTests/ipywidget.ui.functional.test.ts:235]: Oh okay. I thought that was what this error needed to be resolved. I guess it legitimately timed out then.
> Timeout of 180000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

Comment (DonJayamanne) [reply src/test/datascience/uiTests/ipywidget.ui.functional.test.ts:235]: Yup, legitimately timedout.

Comment (DonJayamanne) [reply src/test/datascience/uiTests/ipywidget.ui.functional.test.ts:235]: You'll need to increase the time out of the tests.
However `180000ms` is 3 minutes, thats long enough for individual tests.

Comment (joyceerhl) [reply src/test/datascience/uiTests/ipywidget.ui.functional.test.ts]: Reverted this change. I haven't fixed the ipywidgets tests completely but this branch still contains changes that I think are necessary, so going to merge unless anyone objects

**Pull request merged**